### ISSUE TITLE
Add links to 2018 disability and pension rates pages to bottom of current rates pages

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#43676]
## Issue Description
When we created our modernized rates pages, we began linking to the past 3 years of rates at the bottom of the page as well as in the left nav. We've now been creating current rates pages long enough to have 4 years of modernized rates for many of the disability and pension pages. We had removed the 2018 modernized rates page from the bottom of the current page to make room for 2021 and we took it off of the left nav.

After discussing with Mikki, we are going to add a link to 2018 modernized rates back to the bottom of the current rates page but not include it in the left nav. So now there will be 4 years worth of rates on the bottom of each current disability or pension rates page. Going forward as we create new years of rates, we'll keep adding to the list of past rates on the current page (2022, 2021, 2020, 2019, 2018, etc.). 

**Note:** We only want to add a link to our modernized past rates pages (Drupal pages). We don't need to add any that are on benefits.va.gov.

Pages to add links to:
- [x] [Birth defect rates](https://www.va.gov/disability/compensation-rates/birth-defect-rates/)
- [x] [Special monthly compensation rates](https://www.va.gov/disability/compensation-rates/special-monthly-compensation-rates/)
- [x] [Special benefit allowances rates](https://www.va.gov/disability/compensation-rates/special-benefit-allowance-rates/)
- [x] [DIC rates for parents](https://www.va.gov/disability/parent-dic-rates/)
- [x] [DIC rates for spouses and dependents](https://www.va.gov/disability/survivor-dic-rates/)
- [x] [Veterans disability compensation rates](https://www.va.gov/disability/compensation-rates/veteran-rates/)
- [x] [Survivors Pension rates](https://www.va.gov/pension/survivors-pension-rates/)
- [x] [Protected disability and death pension rates](https://www.va.gov/pension/protected-pension-rates/)
- [x] [Pension rates for Veterans](https://www.va.gov/pension/veterans-pension-rates/)
 
## Tasks
- [x] Add link to 2018 rates to bottom of each rates page listed above using same styling as other links in the last
- [ ] Final copyedit (Megan)
- [ ] Publish (Megan)

## Acceptance Criteria
- [ ] All our modernized past rates pages are linked to from current pages

**Comment by katherine-fung** [comment]
Birth defect rates
https://prod.cms.va.gov/node/993/latest

Special monthly compensation rates
https://prod.cms.va.gov/node/990/

Special benefit allowances rates
https://prod.cms.va.gov/node/991/

DIC rates for parents
https://prod.cms.va.gov/node/1013/latest

DIC rates for spouses and dependents
https://prod.cms.va.gov/node/1012/latest

Veterans disability compensation rates

**Comment by megzehn** [comment]
Thanks, @katherine-fung! 

I'll update this list here when they are copyedited and published:

- [ ] Birth defect rates
https://prod.cms.va.gov/node/993/latest

- [ ] Special monthly compensation rates
 https://prod.cms.va.gov/node/990/

- [ ] Special benefit allowances rates
 https://prod.cms.va.gov/node/991/

- [ ] DIC rates for parents
 https://prod.cms.va.gov/node/1013/latest

- [ ] DIC rates for spouses and dependents
https://prod.cms.va.gov/node/1012/latest
 
- [ ] Veterans disability compensation rates
 https://prod.cms.va.gov/node/986/latest

- [ ] Survivors pension rates
 https://prod.cms.va.gov/node/589/latest
 
- [ ] Protected disability and death pension rates
 https://prod.cms.va.gov/node/5090/latest 

- [ ] Pension rates for Veterans
 https://prod.cms.va.gov/node/591/latest

**Comment by megzehn** [comment]
@katherine-fung and @RLHecht: 

I reviewed these changes and published the ones that are checked off above. 

I held off on the 1st and 3rd pages. We reworked those, and Beth is still reviewing our changes. 

Here are the latest tickets for those, for reference: 

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/43548

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/42773
